app/layouts/bare.vue

```diff
@@ -7,6 +7,6 @@
 
 <script>
 export default {
-  name: 'Bare'
+  name: 'BareLayout'
```

Comment:
@baxgas where you you use `BareLayout`? `error.vue` seems to use `bare` as a layout.

Reply:
@gammaw  it is just some suggestion from the linter, doesn't influence the functionality